src/mx_bluesky/hyperion/experiment_plans/pin_tip_centring_plan.py

```diff
@@ -83,18 +83,18 @@ def pin_tip_valid(pin_xy: Tip):
         if pin_tip_valid(tip_xy_px):
             return (int(tip_xy_px[0]), int(tip_xy_px[1]))
 
-        if tip_xy_px[0] == 0:
-            # Pin is off in the -ve direction
-            step_size_mm = -step_size_mm
+        # Pin is off in the -ve direction
```

Comment:
```suggestion
        # Pin is off in the -ve direction if the returned tip x pixel value is 0
```
Could: Amend this comment to specify which condition suggests a -ve pin direction